services/validators/envValidator.go

```diff
@@ -101,6 +143,7 @@ func ValidateEnv() bool {
 	//create workspace table and insert token
 	createWorkspaceTable()
 	insertTokenIfNotExists()
+	insertWHSchemaVersionIfNotExits()
```

Comment:
If workspace token changes, we should also insert in the new DB after line number 167.